README.md

```diff
@@ -124,28 +124,32 @@ export function detect(source) {
   return source.match(/some-test/i) !== null;
 }
 
-export function parse({source, generateSourceMap}, done) {
-  // Here you convert the source into refract elements.
+export function parse({minim, generateSourceMap, source}, done) {
```

Comment:
@pksunkara using `import minim` or `minimModule.namespace()` is wrong here because we are getting a Minim namespace passed to us from Fury rather than importing the module and using our own namespace. We want the current namespace that is being used by the Fury instance so we don't need to care about which elements are loaded and how they have been loaded.

```js
export function parse({minim, /* ... */}, done) {
  // ...
}

// Is the same as

exports.parse = function parse(options, done) {
  var minim = options.minim;
  // ...
}
```

See [Object Destructuring and Function Argument Defaults](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) from ES6 for more info.